prime-dotnet-webapi/Controllers/EmailsController.cs

```diff
@@ -0,0 +1,39 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Auth;
+using Prime.Services;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/emails")]
+    [ApiController]
+    public class EmailsController : ControllerBase
+    {
+        private readonly IEmailService _emailService;
+
+        public EmailsController(IEmailService emailService)
+        {
+            _emailService = emailService;
+        }
+
+        // GET: api/emails/management/statuses
+        /// <summary>
+        /// Update all logged email statuses sent using the CHES email service
+        /// </summary>
+        [HttpGet("management/statuses", Name = nameof(UpdateEmailLogStatuses))]
+        [Authorize(Roles = AuthConstants.PRIME_API_SERVICE_ACCOUNT_ROLE + "," + AuthConstants.ADMIN_POLICY)]
```

Comment:
As written, this wont allow admins to access this endpoint. I assume this is a typo, you need the Admin role not the admin policy (PRIME_ADMIN_ROLE). 

I have a slight preference for interpolated string for this kind of thing, but its not that much of a difference.

`[Authorize(Roles = $"{AuthConstants.PRIME_API_SERVICE_ACCOUNT_ROLE},{AuthConstants.PRIME_ADMIN_ROLE}")]`